contrib/builder/deb/armhf/generate.sh

```diff
@@ -83,15 +83,15 @@ for version in "${versions[@]}"; do
 	)
 	# packaging for "sd-journal.h" and libraries varies
 	case "$suite" in
-		precise|wheezy) ;;
-		jessie|trusty) packages+=( libsystemd-journal-dev );;
-		*) packages+=( libsystemd-dev );;
+		wheezy) ;;
+		jessie|trusty) packages+=( libsystemd-journal-dev ) ;;
+		*) packages+=( libsystemd-dev ) ;;
```

Comment:
@albers Is the change I made incorrect (I quickly tested it, and it seemed to work the same still), happy to change to what's best 😅